### DISCUSSION TITLE
一堆冰冷的二进制数据，饱含了多少人的心酸与无奈。

### DISCUSSION THREAD

**Discussion by summerBreeze03** [WeNeedHome/SummaryOfLoanSuspension#1206] [General]
有个预感，最近几年 央视、共青团等官方媒体越来越逆天，越来越脱离群众，网络上的风气越来越差，疫情之后网络上的风气大转变。
改革开放以来，确实飞速发展了许多，但万事万物都有代价，弊端也在不断积累，也是时候显现了

**Comment by Cegazhos** [comment]
你應該補習一些真正的歷史，例如你黨自己的黨史。之後 你應該就明白了 從來都不是以人爲本的。

**Reply by summerBreeze03** [reply]
其实我早就知道了，小时候接受了过量的爱国教育，当我成年以后面对真正的世界的那一刻我甚至感觉有点陌生。一味的演示，迟早会毁了一代人。
顺带，可以拜托发给我几个链接吗？我在这里肯定是搜索不到任何信息。

**Reply by Cegazhos** [reply]
什么链接。。。你不想补习历史 看那些研究历史的讲座视频也行嘛。
